.env.default

```diff
@@ -7,6 +7,7 @@ CISCOSPARK_SCOPE="Identity:OAuthClient webexsquare:get_conversation webexsquare:
 SCOPE="Identity:OAuthClient webexsquare:get_conversation webexsquare:admin Identity:SCIM spark:people_read spark:rooms_read spark:rooms_write spark:memberships_read spark:memberships_write spark:messages_read spark:messages_write spark:applications_read spark:applications_write spark:teams_read spark:teams_write spark:team_memberships_read spark:team_memberships_write spark:bots_read spark:bots_write spark:kms"
 COMMON_IDENTITY_SERVICE="spark"
 PORT="8000"
+SKIP_FLAKY_TESTS="true"
```

Comment:
@mattnorris -- opinion?

Reply:
removing it for now. Dont remember adding that for this PR